tests/bluetooth/bsim_bt/bsim_test_multiple/tests_scripts/multiple.sh

```diff
@@ -13,7 +13,7 @@ function Execute(){
  compile it?)\e[39m"
     exit 1
   fi
-  timeout 900 $@ & process_ids="$process_ids $!"
+  timeout 1200 $@ & process_ids="$process_ids $!"
```

Comment:
This timeout is just a safety-stop for a hung tests. Better bump it up to 1800 or more (Some systems out there can be quite slow, no need to keep it tight).

Reply:
sure, I will bump it to 1800.

Reply:
By the way, it is a bit scary that it takes this long in CI.. (see my comment in discord about how long it takes in my machine)
Do you see similar slowdowns in all other compiles/tests?

Reply:
> sure, I will bump it to 1800.

Done

> By the way, it is a bit scary that it takes this long in CI.. (see my comment in discord about how long it takes in my machine)
> Do you see similar slowdowns in all other compiles/tests?

So far, only this test needed changing the timeout value. No idea what causes this test to take so long in the CI; but, if I remember correctly, it did take much longer than ~3 minutes on my local machine as well last time I checked.